python/cuml/benchmark/algorithms.py

```diff
@@ -82,48 +83,65 @@ def __init__(
             self.name = cuml_class.__name__
         self.accepts_labels = accepts_labels
         self.bench_func = bench_func
+        self.setup_cpu_func = setup_cpu_func
+        self.setup_cuml_func = setup_cuml_func
         self.cpu_class = cpu_class
         self.cuml_class = cuml_class
         self.shared_args = shared_args
         self.cuml_args = cuml_args
         self.cpu_args = cpu_args
         self.data_prep_hook = data_prep_hook
         self.accuracy_function = accuracy_function
+        self.cpu_obj = None 
```

Comment:
Since this class abstracts an "algorithm", I wouldn't want its members to include transient data that happens to be in use currently.